hca/upload/checksum_handler.py

```diff
@@ -0,0 +1,67 @@
+import hashlib
+import time
+import zlib
+
+import crcmod
+
+# Checksum(s) to compute for file; current options: crc32, crc32c, sha1, sha256
+CHECKSUM_NAMES = ['crc32']
+
+
+class ChecksumHandler:
```

Comment:
I'd recommend modifying and using https://github.com/HumanCellAtlas/dcplib/tree/master/dcplib/checksumming_io since it is shared across DCP components and will provide a uniform checksumming.

Reply:
Done. https://github.com/HumanCellAtlas/dcplib/pull/6. Will modify this PR afterwards.